ui/components/app/snaps/snap-permissions-list/snap-permissions-list.js

```diff
@@ -7,19 +7,20 @@ import { Box } from '../../../component-library';
 
 export default function SnapPermissionsList({
   snapId,
+  snapName,
   permissions,
-  targetSubjectMetadata,
   showOptions,
 }) {
   const t = useI18nContext();
```

Comment:
Wondering if it would be easier for the SnapPermissionList to get the Snap name from the ID?

Reply:
I kept it like this because the settings page uses the `snapName` quite a bit and I didn't see the sense in recalculating it.